infrastructure/loadtesting/terraform/loadtesting.tf

```diff
@@ -51,7 +51,7 @@ resource "aws_ecs_task_definition" "loadtest" {
         command = [
           "/go/osquery-perf",
           "-enroll_secret", data.aws_secretsmanager_secret_version.enroll_secret.secret_string,
-          "-host_count", "5000",
+          "-host_count", "500",
```

Comment:
This is an intentional change, as we add more functionality to `osquery-perf` we have seen the container OOM.

Reply:
Just a theory, this may be due to not cleaning up some resources. See above on ticker. I would retry after making the change to see if the memory usage decreases.

Reply:
I run another load test after fixing the memory leak, but unfortunately the containers still OOM :(